cocotb/binary.py

```diff
@@ -327,8 +327,15 @@ def signed_integer(self, val):
     get_value_signed = signed_integer.fget
 
     @property
-    def is_resolvable(self):
-        """Does the value contain any ``X``'s?  Inquiring minds want to know."""
+    def is_resolvable(self) -> bool:
+        """Return whether the value contains only resolvable (i.e. no "unknown") bits.
```

Comment:
```suggestion
        """
        Return whether the value contains only resolvable (i.e. no "unknown") bits.
```
Is there an "official" recommendation on this style? I was under the impression keeping them on the same line for a one-liner docstring and having them on different lines for multiline docstrings was the recommended style. But what do I know?

Reply:
It's in https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
>  The summary line may be on the same line as the opening quotes or on the next line.

Seems like it doesn't matter. I think most of the current docstrings use the next line.